concent_api/core/utils.py

```diff
@@ -118,3 +123,34 @@ def extract_name_from_scene_file_path(absoulte_scene_file_path_in_docker: str) -
             ErrorCode.MESSAGE_INVALID
         )
     return relative_scene_file_path_in_archive
+
+
+def is_protocol_verison_compatible_with_verison_in_concent(protocol_version: str) -> bool:
```

Comment:
make it shorter:
`is_protocol_version_compatible` or 'is_golem_messages_version_supported`